lms/config.py

```diff
@@ -53,7 +53,6 @@ def __post_init__(self):
     _Setting("jwt_secret"),
     _Setting("google_client_id"),
     _Setting("google_developer_key"),
-    _Setting("google_app_id"),
```

Comment:
This was already unused, correct? Sounds like it based on [this commit](https://github.com/hypothesis/lms/pull/4732/commits/20e7e2903e1d7f968cd0bb9f3de33acd70389f79)

Reply:
That's correct. This hasn't been used in years.